tlf.Rproj

```diff
@@ -1,8 +1,8 @@
 Version: 1.0
 
-RestoreWorkspace: Default
-SaveWorkspace: Default
-AlwaysSaveHistory: Default
+RestoreWorkspace: No

```

Comment:
@IndrajeetPatil Why those changes?

Reply:
AFAIK, this is considered good practice for long-term reproducibility, e.g. see https://usethis.r-lib.org/reference/use_blank_slate.html. Better to start each session of the package project completely fresh without any residuals from the previous session.

But happy to revert if you see that there is a utility for such behaviour for OSP projects.

Reply:
no no. I have no idea what this means :) Asking for as friend :)

Reply:
Projects using ospsuite-r should NEVER restore the workspace, as it may produce null pointers and crash the session - so at least for ospsuite-r we should add this! (and esqlabsR - @IndrajeetPatil could you do this?)

Reply:
Yepp, did this a couple of days ago! :)

https://github.com/esqLABS/esqlabsR/blob/dc27549036301beeb03e0fbb5a525f40ab8aac3e/esqlabsR.Rproj#L3

Reply:
here for ospsuite-r
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/612